cli/src/runner.rs

```diff
@@ -107,39 +113,10 @@ pub async fn extract_failed_tests(
     repo: &BundleRepo,
     org_slug: &str,
     file_sets: &[FileSet],
-    start: Option<SystemTime>,
 ) -> anyhow::Result<Vec<Test>> {
     let mut failures = Vec::<Test>::new();
     for file_set in file_sets {
         for file in &file_set.files {
-            log::info!("Checking file: {}", file.original_path);
-            let metadata = match metadata(&file.original_path) {
-                Ok(metadata) => metadata,
-                Err(e) => {
-                    log::warn!("Error getting metadata: {}", e);
-                    continue;
-                }
-            };
-            let time = match metadata.modified() {
-                Ok(time) => time,
-                Err(e) => {
-                    log::warn!("Error getting modified time: {}", e);
-                    continue;
-                }
-            };
-            // skip files that were last modified before the test started
-            // this is only used when running tests in wrap mode
-            if let Some(start) = start {
-                if time <= start {
-                    log::info!(
-                        "Skipping file `{}` because modified time `{}` is before start time `{}`",
-                        file.original_path,
-                        DateTime::<Utc>::from(time),
-                        DateTime::<Utc>::from(start),
-                    );
-                    continue;
-                }
-            }
```

Comment:
This moves the timestamp checking into the glob scan as an optional parameter. This way it applies to both upload and test subcommands.